specification/flag-evaluation/hooks.md

```diff
@@ -137,26 +114,23 @@ client.getValue('my-flag', 'defaultValue', new Hook3());
 
 ##### Requirement 4.4.3
 
-> If an error occurs in the `finally` hook, it **MUST NOT** trigger the `error`
-> hook.
+> If a `finally` hook abnormally terminates, evaluation **MUST** proceed normally, including the execution of any remaining `finally` hooks.
 
-In practice, this means that errors that occur in the finally hook will bubble
-up.
+In languages with try/catch semantics, this means that exceptions thrown in `finally` hooks should be caught, and not propagated up the call stack.
 
 ##### Requirement 4.4.4
 
-> If an error occurs in the `before` or `after` hooks, the `error` hooks
-> **MUST** be invoked.
+> If an `error` hook abnormally terminates, evaluation **MUST** proceed normally, including the execution of any remaining `error` hooks.
+
+In languages with try/catch semantics, this means that exceptions thrown in `error` hooks should be caught, and not propagated up the call stack.
```

Comment:
@justinabrahms @beeme1mr Here are the new requirements around errors/finally, as discussed.